Expecto/Expecto.fs

```diff
@@ -243,6 +243,13 @@ module Tests =
       | Focused -> ftestCaseTask name a
       | Pending -> ptestCaseTask name a
 
+  [<AutoOpen>]
+  module TestTaskExtensions =
+    type TestTaskBuilder with
```

Comment:
Adding these overloads as extensions helps the compiler know to prioritize the original Task overload, which prevents breakage in existing code that may have implicitly typed an expression based on Bind originally only taking variations of Task.